src/tests/home.test.js

```diff
@@ -0,0 +1,11 @@
+import renderer from 'react-test-renderer';
+import Home from '../components/Home';
+
+describe('test for home', () => {
+  test('test if home matches', () => {
+    const homeComponent = renderer
+      .create(<Home />)
+      .toJSON();
+    expect(homeComponent).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Please make sure to implement integration tests by using React Testing Library to simulate user interaction, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md#:~:text=Use%20React%20Testing%20Library%20to%20simulate%20user%20interaction.) requirement.
- [ ] Kindly make sure that all tests pass correctly. Currently, there's one failing test.

![image](https://user-images.githubusercontent.com/62290545/235710611-6951422c-c3e6-4fab-a893-592fcfe5d0ec.png)